components/action/GeometricBackground.tsx

```diff
@@ -0,0 +1,17 @@
+import GeometricBackgroundSmall from './GeometricBackgroundSmall';
+import GeometricBackgroundLarge from './GeometricBackgroundLarge';
+
+const GeometricBackground = () => {
+  return (
+    <>
```

Comment:
Use a `<div>` here instead of the fragments.

Reply:
Resolved

Reply:
resolved